modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/new-claim-button/index.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-undef */
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+const currentPath = Liferay.currentURL.split('/');
+const mdfRequestedId = +currentPath[currentPath.length - 1];
```

Comment:
Thinking about the result of our build process, and browser support, is it possible using [`Array.prototype.at()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at)?

Reply:
Hmm, I just noticed this is a fragment. Does fragments code goes through some (e.g.: babel) build processes?